<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoRunPanel</class>
 <widget class="QWidget" name="AutoRunPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AutoRunPanel</string>
  </property>
  <widget class="QTableView" name="tableView_resultList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>361</width>
     <height>350</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_stopAutoRun">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>720</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>停止运行测试</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>360</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>运行状态</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>112</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>进度：</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_resultPrg">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>281</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>75</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>状态:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_resultOutput">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 48pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>NA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_resultStatus">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 127);</string>
    </property>
    <property name="text">
     <string>工位空闲中</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_startAutoRun">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>660</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>启动运行测试</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>360</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>操作运行</string>
   </property>
   <widget class="QPushButton" name="pushButton_runOnce">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>单步测试</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_AutoRun">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>自动运行</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
